client/lib/theme/utils.js

```diff
@@ -0,0 +1,3 @@
+export function isActiveThemeFSEEnabled( activeThemeData ) {
```

Comment:
It would be helpful to have a [doc block](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#functions) for this function that indicates it's meant for reading the response for the active theme from /wp/v2/themes?status=active. I think that would help clarify what it's for, otherwise it's a bit mysterious on its own.

An alternative to consider would be wrapping this logic into a separate higher-order component, itself, so there's no need for an additional util. For example,

```js
const MyComponent = function( { isFSEActive, isFSEActiveLoading } ) {
    ...
}

withIsFSEActive( MyComponent );
```

What do you think?

Reply:
I considered using a higher-order component (HOC), but I did not want to create a more complex nested component structure (Especially with the components that are already wrapped inside another HOC) and figured the utility function would be more versatile/reusable. Since the `isActiveThemeFSEEnabled` function is not related to React or a specific component, I decided to separate it out into its own utility function. Are there any other benefits you can see for using a HOC instead of a util function?

Reply:
> I did not want to create a more complex nested component structure

There's not really any additional nesting required: in my example `withActiveTheme` would be replaced with `withIsFSEActive` and work the same way, just return more specific information.

A middle ground might be adding the `isFSEActive` prop to the `<Wrapper>` component within `withActiveTheme`, so the `isFSEActive` prop is directly available to any components wrapped with `withActiveTheme`.

> Are there any other benefits you can see for using a HOC instead of a util function?

The main benefit I see is it's one less import (because the util function requires an additional import). And since the util function is not likely to be used for anything other than looking at the result of the `/themes?status=active` response, I don't see much benefit to it existing apart from the HOC (at least until we have a specific use case for it).

Reply:
> A middle ground might be adding the isFSEActive prop to the <Wrapper> component within withActiveTheme, so the isFSEActive prop is directly available to any components wrapped with withActiveTheme.

I'll move the util function logic into `withActiveTheme` and add `isFSEActive` prop to the Wrapper component.